README.md

```diff
@@ -103,6 +99,9 @@ We'd recommend Google Chrome or Mozilla Firefox instead.
 
 ![screenshot of phone app](https://s3.amazonaws.com/com.twilio.prod.twilio-docs/images/TwilioClientQuickstart.original.png)
```

Comment:
This app doesnt seem to work for me. I constantly get `Twilio.Device Error: Invalid Access Token issuer/subject`

Reply:
Are you testing locally with the `code-exchange-upgrade` branch? 
Are `TWILIO_ACCOUNT_SID`, `API_KEY`, and `API_SECRET` values correct in your `.env` file?

Reply:
not sure what changed, but this started working for me again. are we correctly setting the environment variables whenever the app is run? or do we expect the user to manually `source .env` after making changes to the file?

Reply:
It's not needed to `source .env`. This change was introduced in this PR.